TabulateSmarterTestContentPackage.sln

```diff
@@ -1,9 +1,11 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Express 2013 for Windows Desktop
-VisualStudioVersion = 12.0.30110.0
+# Visual Studio 14
+VisualStudioVersion = 14.0.24720.0
```

Comment:
Can you see if changing the VS versions back to 12 (which is VS 2013 I think)?  Seems like Smarter is uisng VS 2013 and keeping it the old version should be backward compatible for us using VS 2015.  If it works as expected for us after changing let's leave it that way.